x-pack/plugins/siem/public/components/stat_items/barchart.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  // @ts-ignore
+  EuiSeriesChartUtils,
+} from '@elastic/eui';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+// @ts-ignore
```

Comment:
Doesn't look like this `@ts-ignore` is required. I was able to remove it without error.